### PR TITLE
One day geekweek

### PR DESCRIPTION
**Pull request by Rich-McGrath**
Added an Additional Resources slide and included some prework reading.

### CODE REVIEW

File: one-day-geekweek/agenda.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+# One Day DevOps / Agile / Cloud Native Geek Week
+
+A single day, heavy duty, partially hands-on introduction to discussions around cloud native applications, the methods used to develop them, the ideas behind the architecture and some of the tools in use.
+
+---
+
+## Prework
+
+* Read [Phoenix Project](http://itrevolution.com/books/phoenix-project-devops-book/)
+* Read [Intro to the Command Line](https://www.gotealeaf.com/books/command_line)
```
Comment (mcowger) [inline one-day-geekweek/agenda.md]: Disagree here - don't believe that heavy duty CLI understanding is important here.  If you want to find something shorter/smaller I'd reconsider it.

File: one-day-geekweek/agenda.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+* Read [A Guide to Git and GitHub](https://www.gotealeaf.com/books/git)
```
Comment (mcowger) [inline one-day-geekweek/agenda.md]: Definitely not covering this in prework - I do think however its reasonable for 'Additional resources' section.

File: one-day-geekweek/agenda.md [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+* Install `git`
+* Install `cf`
+* Watch [Program or Be Programmed](https://www.youtube.com/watch?v=imV3pPIUy1k)
```
Comment (mcowger) [inline one-day-geekweek/agenda.md]: Don't see the value here, especially of this very religious centric piece.

Comment (Rich-McGrath) [inline one-day-geekweek/agenda.md]: Religious?

**Comment by jonasrosland** [comment]
Closing this based on @mcowger's comments.

**Pull request closed without merging**